lib/renderer/content-scripts-injector.js

```diff
@@ -28,14 +28,26 @@ const runContentScript = function (extensionId, url, code) {
 const injectContentScript = function (extensionId, script) {
   if (!script.matches.some(matchesPattern)) return
 
-  for (const {url, code} of script.js) {
-    const fire = runContentScript.bind(window, extensionId, url, code)
-    if (script.runAt === 'document_start') {
-      process.once('document-start', fire)
-    } else if (script.runAt === 'document_end') {
-      process.once('document-end', fire)
-    } else if (script.runAt === 'document_idle') {
-      document.addEventListener('DOMContentLoaded', fire)
+  if (script.js) {
+    for (const {url, code} of script.js) {
+      const fire = runContentScript.bind(window, extensionId, url, code)
+      if (script.runAt === 'document_start') {
+        process.once('document-start', fire)
+      } else if (script.runAt === 'document_end') {
+        process.once('document-end', fire)
+      } else if (script.runAt === 'document_idle') {
+        document.addEventListener('DOMContentLoaded', fire)
+      }
+    }
+  }
+
+  if (script.css) {
+    for (const {code} of script.css) {
+      process.once('document-end', () => {
+        var node = document.createElement('style')
+        node.innerHTML = code
```

Comment:
Might want to use `node.textContent = code` instead to ensure child elements aren't created.

Reply:
Sounds like a good suggestion, do you want to create a PR for it?